spec/requests/users_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns a success response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get users_path
+      expect(response.body).to include('Here is list of all the users')
+    end
+  end
+
+  describe 'GET /users/:id' do
+    let(:user) { User.create(name: 'Amelia', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from USA.', posts_counter: 0) }
+
+    it 'returns a success response' do
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get user_path(user)
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get user_path(user)
+      expect(response.body).to include('User Details')
+    end
+  end
+end
```

Comment:
- [ ] Hi @aima98, your tests are not running successfully caused by the duplicate user references in the likes table as you can see on the screenshot below:
![Screenshot from 2023-09-26 12-19-11](https://github.com/aima98/blog/assets/63087491/d42b0214-9298-49c3-976f-6a657870ca5a)